server/belga/io/feed_parsers/belga_anpa.py

```diff
@@ -61,15 +61,18 @@ def parse(self, file_path, provider=None):
                 b'([0-9]{1,2})-([0-9]{1,2}) ([0-9]{4})',
                 lines[1], flags=re.I)
             if m:
+                item['language'] = 'en'
+                item['priority'] = 2 if m.group(1).decode() == 'u' else 3
```

Comment:
Is there any ticket(s) regarding changes for this parser?

Reply:
Yes, It like in file groovy, they use groovy file to mapping data in belga360.
you can check in https://dev.sourcefabric.org/browse/SDBELGA-151
file `kyodoText2newsml.groovy`

Reply:
Belga's team tell me make the mapping like xsl, groovy file. The descriptions in this task is only a part of mapping, it use for guide.
note: description mapping AFP product via keywords that is wrong, in xsl it mapping with category.(they raise this bug)